clients/admin-ui/src/features/dataset/types.ts

```diff
@@ -3,6 +3,9 @@ import {
   FidesBase,
   FidesKey,
 } from "~/features/common/fides-types";
+import { DatasetCollection, DatasetField } from "~/types/api";
+
+export type { DatasetCollection, DatasetField } from "~/types/api";
```

Comment:
This whole file can probably be replaced with the generated types, but this gets the ball rolling with some types that are verbose to keep in sync with the API.

Reply:
nice, yeah can we add a ticket to replace these types with the generated types?